tests/test.py

```diff
@@ -3488,24 +3247,6 @@ def test_read_keys_symmetric_zone(self):
             env={ACRA_MASTER_KEY_VAR_NAME: self.master_key},
             timeout=PROCESS_CALL_TIMEOUT)
 
-    def test_keys_generation_for_mixed_keys(self):
-        with self.assertRaises(subprocess.CalledProcessError) as exc:
-            subprocess.check_output([
-                './acra-keys',
-                'generate',
-                '--acraserver_transport_key',
-                '--acratranslator_transport_key',
-                '--poison_record_keys',
-                '--audit_log_symmetric_key',
-                '--keys_dir={}'.format(self.dir_with_distinguished_name_client_id.name),
-                '--keys_dir_public={}'.format(self.dir_with_distinguished_name_client_id.name),
-                '--keystore={}'.format(KEYSTORE_VERSION),
-            ],
-                env={ACRA_MASTER_KEY_VAR_NAME: self.master_key},
-                stderr=subprocess.STDOUT)
-        self.assertIn("--client_id or --tls_cert is required to generate keys".lower(), exc.exception.output.decode('utf8').lower())
```

Comment:
hm, why you removed this test? as I understand we test there requirement to pass `client_id` or `tls_cert` arguments

Reply:
Not exactly, you are right that this test was used to check requirement to pass `client_id` or `tls_cert`, but now threre are no                 '--acratranslator_transport_key' and `--acraserver_transport_key` cli args for `acra-keys generate` so there in no sense for this test in such case because `poison_record_keys` args doesnt require `clien_id` or `tls_cert`, am I right?

Reply:
But we already have such test

https://github.com/cossacklabs/acra/blob/master/tests/test.py#L3421

Here is the souce test we have, without passing transport keys we will be able to test such requirement.